File: public/phase-2-20160620022203/JAVA/arrays-minmax.java
// Name : Arrays Minmax
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n : size of a
		// m : size of b
		// a : array A
		// b : array B
		int index;
		Scanner in = new Scanner(System.in);
		String[] elements = (in.nextLine()).split(" ");
		int n = Integer.parseInt(elements[0]);
		int m = Integer.parseInt(elements[1]);
		int[] a = new int[n];
		String[] a_elements = (in.nextLine()).split(" ");
		for(index=0;index<n;index++)
			a[index] = Integer.parseInt(a_elements[index]);
		int[] b = new int[m];
		String[] b_elements = (in.nextLine()).split(" ");
		for(index=0;index<m;index++)
			b[index] = Integer.parseInt(b_elements[index]);


		// write your code here
		// store your results in `minimum_num_of_operations`

		// output
		// Dummy Data
		int minimum_num_of_operations=3;

		System.out.println(minimum_num_of_operations);

	}
}

File: public/phase-2-20160620022203/JAVASCRIPT/arrays-minmax.js
// Name : Arrays Minmax
// input:
// n : size of a
// m : size of b
// a : array A
// b : array B
var index,minimum_num_of_operations;
var elements = readline().split(" ");
var n = parseInt(elements[0], 10);
var m = parseInt(elements[1], 10);
var a = [];
var a_elements = readline().split(' ');
for(index=0;index<n;index++)
	a.push(parseInt(a_elements[index], 10));
var b = [];
var b_elements = readline().split(' ');
for(index=0;index<m;index++)
	b.push(parseInt(b_elements[index], 10));


// write your code here
// store your results in `minimum_num_of_operations`

// Dummy Data
minimum_num_of_operations = 3;


// output
print(minimum_num_of_operations);
